src/compass/utils/h5_helpers.py

```diff
@@ -545,6 +545,13 @@ def metadata_to_h5group(parent_group, burst, cfg, save_noise_and_cal=True,
     for meta_item in vrt_items:
         add_dataset_and_attrs(vrt_group, meta_item)
 
+    # Fill in the center coordinated with NaN when
+    # the burst does not have center point information
+    if burst.center.is_empty:
+        xy_center = [float('nan')] * 2
+    else:
+        xy_center = [xy[0] for xy in burst.center.coords.xy]
+        
```

Comment:
Just a thought. Do you think there is a way to inform the users that the center is (NaN, NaN) because of missing metadata in the original input granule? I do not want the users to believe that this is the result of a glitch of our processor.

Reply:
This change was reverted. Considering this as resolved because we are not going to take care of missing polygon issue for SET correction.